partiql-cli/src/main/kotlin/org/partiql/cli/Main.kt

```diff
@@ -237,7 +247,42 @@ internal class MainCommand : Runnable {
             // val connector = LocalPlugin.create(root.toPath())
             // return mapOf("default" to connector)
         }
+
+        if (env != null) {
+            return listOf(parsePartiQL(env!!))
+        }
+
         // Derive a `default catalog from stdin (or file streams)
+        return listOf(parseIon())
+    }
+
+    private fun parsePartiQL(env: File): Catalog {
+        val stream = env.inputStream()
+        val compiler = PartiQLCompiler.standard()
+        val parser = PartiQLParser.standard()
+        val planner = PartiQLPlanner.standard()
+        val data = stream.bufferedReader(charset("UTF-8")).use { it.readText() }
+        val parseResult = parser.parse(data)
+        val statement = parseResult.statements[0]
+        val plan = planner.plan(statement, Session.empty()).plan
+        val datum = compiler.prepare(plan, Mode.PERMISSIVE()).execute()
```

Comment:
We should include some proper error messaging if the data loading fails (e.g. if the input PartiQL literal expression is malformed and fails to parse/evaluate).

Reply:
I think we should still output the error from running the parser + evaluator but also include info that the error is coming from the environment file loading. Otherwise, it can be confusing where/how to fix the error.